soc/realtek/ec/rts5912/reg/reg_adc.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2025 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_REALTEK_RTS5912_REG_ADC_H
+#define ZEPHYR_SOC_REALTEK_RTS5912_REG_ADC_H
+
+typedef struct {
+	volatile uint32_t CTRL;
+	volatile uint32_t CHCTRL;
+	volatile uint32_t STS;
+	volatile uint32_t CHDATA[12];
+	volatile uint32_t COEFFA;
+	volatile uint32_t COEFFB;
+} ADC_Type;
+
+/* CTRL */
+#define ADC_CTRL_EN             BIT(0)
+#define ADC_CTRL_START_Pos      (1U)
+#define ADC_CTRL_START_Msk      BIT(ADC_CTRL_START_Pos)
```

Comment:
Simplify all the single-bit definitions.
```suggestion
#define ADC_CTRL_START      BIT(1)
```

Reply:
Fixed